lib/concatSeries.js

```diff
@@ -6,25 +6,19 @@ import doSeries from './internal/doSeries';
 /**
  * The same as `concat` but runs only a single async operation at a time.
  *
+ * @name concatSeries
  * @static
- * @memberof async
+ * @memberOf async
+ * @see async.concat
```

Comment:
Thoughts on using @see over lodashs method. In lodash docs as I understand, anything in code tags with the same @memberOf tag automagically links to that method.

So it would be something like
> see `async.concat`

I'm fine with either approach, lodash's just seems a bit more flexible when multiple methods are linked
/cc @aearly @jdalton